mungegithub/mungers/nag-flake-issues.go

```diff
@@ -128,10 +129,16 @@ func (NagFlakeIssues) Munge(obj *mgh.MungeObject) {
 }
 
 // StaleComments returns a slice of stale comments
-func (NagFlakeIssues) StaleComments(obj *mgh.MungeObject, comments []*github.IssueComment) []*github.IssueComment {
+func (NagFlakeIssues) StaleComments(obj *mgh.MungeObject, issueComments []*github.IssueComment) []*github.IssueComment {
```

Comment:
Sure, I'll do that. I didn't want to change the return type of StaleComments since it is matching an interface that is used in a lot of places and is used to delete IssueComments only (not review comments or reviews themselves). Changing the name to make it clear that the interface function only handles IssueComments makes the most sense. If we ever want to add support for deleting stale review comments it would be easy to add another function to the interface (e.g. StaleReviewComments) and update the comment_deleter munger to delete review comments as well.